sunpy/coordinates/utilities.py

```diff
@@ -0,0 +1,208 @@
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import, division
+import datetime
+import warnings
+
+import numpy as np
+import astropy.units as u
+from astropy.time import Time
+from astropy.coordinates import SkyCoord, Angle, Longitude, PrecessedGeocentric, AltAz, \
+                                get_body_barycentric
+from astropy.coordinates.representation import CartesianRepresentation, SphericalRepresentation
+from astropy._erfa.core import ErfaWarning
+
+from sunpy.time import parse_time
+
+from .frames import HeliographicStonyhurst as HGS
+from .transformations import _sun_detilt_matrix
+
+__all__ = ['get_earth', 'get_sun_B0', 'get_sun_L0', 'get_sun_P', 'get_sunearth_distance',
+           'get_sun_orientation']
```

Comment:
is it worth adding a wrapper for `get_body_barycenter` but where it returns a HGS frame?